src/command_modules/azure-cli-extension/azure/cli/command_modules/extension/_params.py

```diff
@@ -2,21 +2,34 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
+import argparse
 
 from argcomplete.completers import FilesCompleter
 
+from azure.cli.core.util import CLIError
 from azure.cli.core.sdk.util import ParametersContext
 from azure.cli.core.extension import get_extension_names
+from ._index import get_index_extensions
 
 
 def extension_name_completion_list(prefix, **kwargs):  # pylint: disable=unused-argument
     return get_extension_names()
 
 
+def extension_name_from_index_completion_list(prefix, **kwargs):  # pylint: disable=unused-argument
```

Comment:
Replace `kwargs` with `_` can save this pylint disable switch.

Reply:
All other completers have `(prefix, **kwargs)` so would like to stick to that.